### ISSUE TITLE
AP for custom datasets

### ISSUE THREAD

**Issue by timscorbett** [xingyizhou/CenterNet#888]
How can I compute accuracy for custom COCO formatted datasets?

**Comment by xingyizhou** [comment]
Hi,
  Please refer to how to add a new dataset in our [document](https://github.com/xingyizhou/CenterNet/blob/master/readme/DEVELOP.md#new-dataset). Feel free to followup if you have a specific question.

**Comment by timscorbett** [comment]
Thank you, the following produced the AP estimates after training.

['src/main.py', 'ctdet', '--dataset', 'ML', '--exp_id', 'ML', '--batch_size', '32', '--num_epochs', '150', '--keep_res', '--test', '--load_model', '/centernet/exp/ctdet/ML/model_best.pth']

I would like to get AP estimates for every 10 epochs or so. Is there a command line argument that produces model_* for regular intervals of epochs? Or is there an alternative way?

Thank you!
